sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingJoinSuite.scala

```diff
@@ -777,6 +781,35 @@ class StreamingOuterJoinSuite extends StreamingJoinSuite {
         CheckNewAnswer(outerResult),
         assertNumStateRows(total = 2, updated = 1)
       )
+
+      Seq(
+        ("10 minutes",
+          "interval 3 minutes 30 seconds"),
+        ("10 minutes",
```

Comment:
We should support watermarks as ANSI intervals in the future. @sarutak Could you open an JIRA for that, so, we won't forget to implement it.

Reply:
O.K, I'll open it.